src/index.js

```diff
@@ -7,20 +7,25 @@ import "../src/index.css";
 import ProductList from "./pages/productListing/ProductList";
 import Login from "./pages/Login/Login";
 import Signup from "./pages/Signup/Signup";
+import Wishlist from "./pages/wishlist/Wishlist";
+import { ProductProvider } from "./Context/product-context";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <BrowserRouter>
-      <Routes>
-        <Route path="/" element={<App />} />
-        <Route path="/productlist" element={<ProductList />} />
-        <Route path="/login" element={<Login />} />
-        <Route path="/signup" element={<Signup />} />
-      </Routes>
-    </BrowserRouter>
+    <ProductProvider value={{ item: 5 }}>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<App />} />
+          <Route path="/productlist" element={<ProductList />} />
+          <Route path="/login" element={<Login />} />
+          <Route path="/signup" element={<Signup />} />
+          <Route path="/wishlist" element={<Wishlist />} />
+        </Routes>
```

Comment:
Just curious, why have you included the routes in index.js and not in App.js

Reply:
No particular reason actually I was doing scrimba previously and they had done it this way, would it better to include them in App.js ?